corehq/apps/reports/filters/users.py

```diff
@@ -352,6 +352,24 @@ def user_es_query(cls, domain, mobile_user_and_group_slugs, request_user):
         location_ids = cls.selected_location_ids(mobile_user_and_group_slugs)
 
         user_type_filters = []
+        has_user_ids = False
```

Comment:
You could get away with this var as it's just `len(user_ids) > 0`. Or if you want to use that var name, set the value here directly and use it under the [below condition](https://github.com/dimagi/commcare-hq/pull/29015/files#diff-bc46e3252dc779e4c1d5df39af6e1bed04a9356e6ad361a3fc0460170bdbbbeeR357) as well.

Reply:
Done.